qualtran/drawing/qpic_diagram_test.py

```diff
@@ -65,20 +65,20 @@ def test_qpic_data_for_reflect_using_prepare():
 reg_2 / \textrm{\scalebox{0.5}{QAny(2)}}
 reg_3:on G:width=25:shape=box \textrm{\scalebox{0.8}{alloc}}
 reg_3 / \textrm{\scalebox{0.5}{QAny(1)}}
+selection G:width=121:shape=box \textrm{\scalebox{0.8}{StatePreparationAliasSampling}} reg G:width=37:shape=box \textrm{\scalebox{0.8}{sigma\_mu}} reg_1 G:width=17:shape=box \textrm{\scalebox{0.8}{alt}} reg_2 G:width=21:shape=box \textrm{\scalebox{0.8}{keep}} reg_3 G:width=65:shape=box \textrm{\scalebox{0.8}{less\_than\_equal}}
```

Comment:
should we scrap this test? I guess it sortof forces you to look at the circuit so you can paste in the new data but it's not very DAMP https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html

Reply:
removed